pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/TransformFunctionFactory.java

```diff
@@ -82,6 +82,7 @@ private TransformFunctionFactory() {
           put(canonicalize(TransformFunctionType.LN.getName().toLowerCase()), LnTransformFunction.class);
           put(canonicalize(TransformFunctionType.SQRT.getName().toLowerCase()), SqrtTransformFunction.class);
 
+          put(canonicalize(TransformFunctionType.AS.getName().toLowerCase()), AsTransformFunction.class);
```

Comment:
Can we set it up so that user won't be able to invoke AS function directly from SQL, so that they use the AS keyword?

Reply:
no, this is used universally can not be changed